src/general/g.citation/g.citation.py

```diff
@@ -186,24 +189,30 @@ def clean_line_item(text):
     return text
 
 
-def get_year_from_documentation(text):
-    """Extract year from text containing SVN date entry
-
-    >>> text = "<p><i>Last changed: $Date: 2011-09-29 15:18:47 $</i>"
-    >>> get_year_from_documentation(text)
-    2011
+def get_datetime_from_documentation(text):
+    """Extract time of latest change from manual
+    >>> text = "  Latest change: Monday Jun 28 11:54:09 2021 in commit: 1cfc0af029a35a5d6c7dae5ca7204d0eb85dbc55"
+    >>> get_datetime_from_documentation(text)
+    datetime.datetime(2022, 9, 18, 23, 55, 9)
     """
-    # we try to capture even when not properly worded (same below)
-    # offending modules: grep -IrnE '\$Date: ' | grep -v "Last changed:"
-    year_capture = (
-        r"<p>\s*<(i|em)>(Last changed: )?\$Date: ([\d]+)-\d\d-\d\d .*\$</(i|em)>"
-    )
-    match = re.search(year_capture, text, re.MULTILINE | re.DOTALL | re.IGNORECASE)
+    datetime_capture = r"^  (Latest change: )(.*)( in commit: ).*"
+    match = re.search(datetime_capture, text, re.MULTILINE | re.DOTALL | re.IGNORECASE)
     if match:
-        return int(match.group(3))
+        date_format = "%A %b %d %H:%M:%S %Y"
     else:
+        datetime_capture = r"^  (Accessed: ).*([A-Z][a-z]{2} [A-Z][a-z]{2}\s+[0-9]{1,2} [0-9]{2}:[0-9]{2}:[0-9]{2} [0-9]{4}).*"
+        match = re.search(
+            datetime_capture, text, re.MULTILINE | re.DOTALL | re.IGNORECASE
+        )
+        date_format = "%a %b %d %H:%M:%S %Y"
```

Comment:
This line is no needed anymore. Please, remove it.

Reply:
Updated in [6042699](https://github.com/OSGeo/grass-addons/pull/808/commits/6042699713f47fe895271a42ede031d5ee8d1a28) accordingly...